examples/learning/miner/2-conjunct/2-conjunct.scm

```diff
@@ -45,20 +46,15 @@
 ;;
 ;; (Lambda
 ;;   (VariableList
-;;     (Variable "$X")
-;;     (Variable "$Y")
-;;     (Variable "$Z")
-;;     (Variable "$W"))
+;;     (Variable "$X-0")
+;;     (Variable "$X-1"))
 ;;   (And
 ;;     (Inheritance
-;;       (Variable "$X")
-;;       (Variable "$Y"))
-;;     (Inheritance
-;;       (Variable "$Z")
-;;       (Variable "$W"))))
+;;       (Variable "$X-0")
+;;       (Variable "$X-1"))))
```

Comment:
Actually `(conjunct-pattern 2)` merely produces a binary conjunction, so indeed the comment was wrong, but the correct answer is to remove the `InheritanceLink` at all.